lib/controller/UserController.js

```diff
@@ -21,17 +21,58 @@ const UserController = [];
  * @name getUsers
  * @type {Hapi.ServerRoute}
  */
+// UserController.push({
```

Comment:
@imadsyed333 - can you delete this if we don't need it anymore

Reply:
Sure!